readme.rst

```diff
@@ -305,7 +306,20 @@ get started, we will:
 Install prerequisite software
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To install the prerequisites:
+Before you start, make sure you have these command-line tools available:
+
+- ``make`` to handle ``Makefile`` recipes
+- ``python3`` to run Sphinx
+- ``apt``, if the ``venv`` Python module isn't available
```

Comment:
1. i feel like we can assume `apt` is installed
2. how is `apt` related to the `venv` module?
3. how likely is it that `venv` would not be available if `python3` is installed?

i fear that we may now be overdoing the requirements aspect

Reply:
This would make `snapd` and `npm` a requirement for the starter pack again, which apparently is controversial.
They should only be required for running local tests.